boards/pimoroni-tufty2040/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "pimoroni-tufty2040"
+version = "0.1.0"
+authors = ["The rp-rs Developers"]
+edition = "2018"
+homepage = "https://github.com/rp-rs/rp-hal-boards/tree/main/boards/pimoroni-tufty2040"
+description = "Board Support Package for the Pimoroni Tufty2040"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rp-rs/rp-hal-boards.git"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cortex-m.workspace = true
+cortex-m-rt = { workspace = true, optional = true }
+embedded-hal.workspace = true
+fugit.workspace = true
+rp2040-boot2 = { workspace = true, optional = true }
+rp2040-hal.workspace = true
+st7789.workspace = true
+display-interface.workspace = true
+embedded-graphics.workspace = true
+pio.workspace = true
+pio-proc.workspace = true
+
+[dev-dependencies]
+panic-halt.workspace = true
+nb.workspace = true
+
+[features]
+# This is the set of features we enable by default
+default = ["boot2", "rt", "critical-section-impl", "rom-func-cache"]
```

Comment:
tufty2040 is new enough that it should be safe to always enable rom-v2-intrinsics